### DISCUSSION TITLE
Need more information on this behaviour of polygon edge v0.6.0

### DISCUSSION THREAD

**Discussion by ex-shubham-koli** [0xPolygon/polygon-edge#772] [Q&A]
After updating to v0.6.0 I am frequently seeing the 
following logs. as this process is started no new blocks are minted.
any information on this would be helpful.

> 0|node1  | 2022-09-29T06:14:05.062Z [INFO]  polygon.blockchain: block already inserted: block=170997 source=syncer
0|node1  | 2022-09-29T06:14:17.054Z [INFO]  polygon.ibft.consensus: round timeout expired: round=0
0|node1  | 2022-09-29T06:14:17.055Z [INFO]  polygon.ibft.consensus: round started: round=1
0|node1  | 2022-09-29T06:14:39.057Z [INFO]  polygon.ibft.consensus: round timeout expired: round=1
0|node1  | 2022-09-29T06:14:39.059Z [INFO]  polygon.ibft.consensus: round started: round=2
0|node1  | 2022-09-29T06:15:21.060Z [INFO]  polygon.ibft.consensus: round timeout expired: round=2
0|node1  | 2022-09-29T06:15:21.062Z [INFO]  polygon.ibft.consensus: round started: round=3
0|node1  | 2022-09-29T06:16:43.064Z [INFO]  polygon.ibft.consensus: round timeout expired: round=3
0|node1  | 2022-09-29T06:16:43.067Z [INFO]  polygon.ibft.consensus: round started: round=4
0|node1  | 2022-09-29T06:16:43.068Z [INFO]  polygon.ibft.consensus: we are the proposer

![image](https://user-images.githubusercontent.com/53253101/192953906-addca54f-8de8-4fa3-a0b2-11f4c903060a.png)

**Comment by ivanbozic21** [comment]
Hi @shubham-edx,

Can you please try to restart each of your validators one more time and let us know if this behavior will continue?

**Reply by ex-shubham-koli** [reply]
restarted it.
still the same behavior.
this time it went upto 10 rounds.
![image](https://user-images.githubusercontent.com/53253101/193244511-583fb48a-67f4-4336-adcb-36239f721e69.png)

**Reply by ex-shubham-koli** [reply]
One of the validator has these logs.
![image](https://user-images.githubusercontent.com/53253101/193245712-3d2bb8a3-28ca-4dbe-8eee-af56f5239313.png)

**Comment by laviniat1996** [comment]
Hi @shubham-edx ,

Is this PoA or PoS? How many validators and bootnodes did you start your setup with? Are you running this locally or on cloud? Also, did you notice that this happens when you try to do a certain action (like sending a tx), or randomly?

**Reply by ex-shubham-koli** [reply]
@laviniat1996 ,
This POS network has 8 validators which are running on 4 separate AWS instances. 
I am sending mock tx from one account to random accounts. roughly 1 tx every 1 minute.?
if this is due to tx-pool DDos is there any way to empty tx pool?

**Reply by laviniat1996** [reply]
@shubham-edx 
<img width="271" alt="image" src="https://user-images.githubusercontent.com/103824766/193253648-21b5082a-c5a3-43d1-bf89-5327fbf3b96d.png">

It looks like you have 12 validators. Did you stake from other accounts?  If you have staked tokens in the staking contract, but those addresses are not validators (so you don’t have a node running for them), the chain will not work properly.

**Reply by ex-shubham-koli** [reply]
Yes, correct. out of 12 validators, only 8 validators are running remaining 4 validators are not functional. I was under impression that at least 4 validators are enough for consensus to meet and blocks to be produced. this is not the case?

**Comment by laviniat1996** [comment]
@shubham-edx  When you did the setup, did you start the network with 12 validators? Or only 8?

**Reply by ex-shubham-koli** [reply]
I started it with 4. The other 8 were added via staking contract. out of those 8 validators, 4 are not running. 
(the network was created at v0.5 then it was upgraded to 5.1 then yesterday it was upgraded to 0.6)

**Reply by laviniat1996** [reply]
Did you start seeing these issue for the first time when upgrading to v0.6.0?

**Reply by ex-shubham-koli** [reply]
Yes correct.

**Reply by laviniat1996** [reply]
@shubham-edx If so, please make sure you fork the chain, because BLS is not compatible with the old release. You can find how to do it here: https://wiki.polygon.technology/docs/edge/consensus/bls/#3-add-fork-setting .
